lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -3218,8 +3218,12 @@ public String getReleaseString(SimplePageItem i) {
 		     return null;
 		 if (page.isHidden())
 		     return messageLocator.getMessage("simplepage.hiddenpage");
-		 if (page.getReleaseDate() != null && page.getReleaseDate().after(new Date()))
-		     return messageLocator.getMessage("simplepage.pagenotreleased");
+		 if (page.getReleaseDate() != null && page.getReleaseDate().after(new Date())) {
+			 Date releaseDate = page.getReleaseDate();
+			 SimpleDateFormat dateFormat = new SimpleDateFormat("EEE M/d/yy");
```

Comment:
Note that I haven’t accepted this patch. I’m about a week away from looking at new features. I’ll make sure everything is properly internationalized.

On Mar 30, 2016, at 4:23:43 PM, Sam Ottenhoff <notifications@github.com<mailto:notifications@github.com>> wrote:


In lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java<https://github.com/sakaiproject/sakai/pull/2020#discussion_r57957072>:

> @@ -3218,8 +3218,12 @@ public String getReleaseString(SimplePageItem i) {
>                    return null;
>                if (page.isHidden())
>                    return messageLocator.getMessage("simplepage.hiddenpage");
> -              if (page.getReleaseDate() != null && page.getReleaseDate().after(new Date()))
> -                  return messageLocator.getMessage("simplepage.pagenotreleased");
> +              if (page.getReleaseDate() != null && page.getReleaseDate().after(new Date())) {
> +                      Date releaseDate = page.getReleaseDate();
> +                      SimpleDateFormat dateFormat = new SimpleDateFormat("EEE M/d/yy");


The rest of the world doesn't use M/d ... is there some magic internationalization happening anywhere here?

—
You are receiving this because you were assigned.
Reply to this email directly or view it on GitHub<https://github.com/sakaiproject/sakai/pull/2020/files/3095649894b2830efcaf679d4d1f7fd3fad2c78f#r57957072>